docs/hardware/3-esp32/2-quickstart/1-overview.md

```diff
@@ -5,7 +5,7 @@ slug: /hardware/esp32/quickstart
 ---
 
 :::caution
-You must first follow the [Golioth Platform Quickstart](/services/getting-started/) before attempting this guide.
+You must first follow the [Golioth Platform Quickstart](/getting-started/) before attempting this guide.
```

Comment:
My first trip through this I actually got confused whether I was doing the quickstart or the getting started guide. Then I realized they were one and the same eventually.